Makefile

```diff
@@ -17,7 +17,7 @@ test-integ: test
 
 version ?= latest
 test-docker:
-	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/HEAD/prism/prism.sh -o prism.sh
+	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/sendgrid-java-cert/prism/prism-java.sh -o prism.sh
```

Comment:
Should still be using `HEAD`. Or is this just to verify the branch works first?

Reply:
Yeah still verifying